### PR TITLE
Module 14 "Automated testing of Mobile Applications"

### PR DESCRIPTION
**Pull request by VVP65**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,12 +5,57 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.example</groupId>
-    <artifactId>mobile_testing</artifactId>
+    <artifactId>mobileFromStetch</artifactId>
```
Comment (olelysieev-clgx) [inline pom.xml]: Scratch ;)

File: src/main/java/service/DriverSetup.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import java.net.URL;
+
+public class DriverSetup {
+    protected static AndroidDriver driver;
+
+    protected static void prepareDriverNative() throws MalformedURLException {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability("deviceName", "emulator-5554");
+        capabilities.setCapability("platformName", "Android");
+        File appDir = new File("C:\\Users\\Volodymyr_Protsiv\\Desktop\\JavaMentoringProgram\\M15_Automated Testing of Mobile Applications with Java\\mobile-testing\\src\\main\\resources");
```
Comment (olelysieev-clgx) [inline src/main/java/service/DriverSetup.java]: Hardcoded values?

Comment (VVP65) [reply src/main/java/service/DriverSetup.java:17]: fixed

File: pom.xml (hunk 1 above)
Comment (VVP65) [reply pom.xml]: fixed

File: src/main/java/service/DriverSetup.java (hunk 2 above)
Comment (VVP65) [reply src/main/java/service/DriverSetup.java]: fixed

Comment (olelysieev-clgx) [reply src/main/java/service/DriverSetup.java]: Create private static String APPLICATION_PATH with valid path value
Remove line 17
In line 18 use your created APPLICATION_PATH variable except appDir

**Review by olelysieev-clgx — APPROVED** [review]

**Pull request merged**